README.md

```diff
@@ -45,7 +45,7 @@ A diagram of this running 5 steps on a single sample image may help explain how
 
 ## Installation
 
-This library works in the browser, in Node, and on the commandline (CLI), which we think is great. 
+This library works in the browser, in Node, and on the commandline (CLI), which we think is great. You can start a local environement to test te UI with `npm start`
```

Comment:
Oops! Typo - test "the" UI -- otherwise great!!!

Reply:
@jywarren fixed this!!😁
Thanks